examples/sktime/test_sktime_model_export.py

```diff
@@ -0,0 +1,520 @@
+        yield bucket_name
+
+
+@pytest.fixture
+def sktime_custom_env(tmp_path):
+    """Create a conda environment and returns path to conda environment yml file."""
+    conda_env = tmp_path.joinpath("conda_env.yml")
+    _mlflow_conda_env(conda_env, additional_pip_deps=["sktime"])
+    return conda_env
+
+
+@pytest.fixture(scope="module")
+def test_data_airline():
+    """Create sample data for univariate model without exogenous regressor."""
+    return load_airline()
+
+
+@pytest.fixture(scope="module")
+def test_data_longley():
+    """Create sample data for univariate model with exogenous regressor."""
+    y, X = load_longley()
+    y_train, y_test, X_train, X_test = temporal_train_test_split(y, X)
+    return y_train, y_test, X_train, X_test
+
+
+@pytest.fixture(scope="module")
+def auto_arima_model(test_data_airline):
+    """Create instance of fitted auto arima model."""
+    return AutoARIMA(sp=12, d=0, max_p=2, max_q=2, suppress_warnings=True).fit(
+        test_data_airline, fh=[1, 2, 3]
+    )
+
+
+@pytest.fixture(scope="module")
+def naive_forecaster_model_with_regressor(test_data_longley):
+    """Create instance of fitted naive forecaster model."""
+    y_train, _, X_train, _ = test_data_longley
+    model = NaiveForecaster()
+    return model.fit(y_train, X_train, fh=[1, 2, 3])
+
+
+@pytest.mark.parametrize("serialization_format", ["pickle", "cloudpickle"])
+def test_auto_arima_model_save_and_load(auto_arima_model, model_path, serialization_format):
+    """Test saving and loading of native sktime auto_arima_model."""
+    flavor.save_model(
+        sktime_model=auto_arima_model,
+        path=model_path,
+        serialization_format=serialization_format,
+    )
+    loaded_model = flavor.load_model(
+        model_uri=model_path,
+    )
+
+    np.testing.assert_array_equal(auto_arima_model.predict(), loaded_model.predict())
+
+
+@pytest.mark.parametrize("serialization_format", ["pickle", "cloudpickle"])
+def test_auto_arima_model_pyfunc_output(auto_arima_model, model_path, serialization_format):
+    """Test auto arima prediction of loaded pyfunc model."""
+    auto_arima_model.pyfunc_predict_conf = {
+        "predict_method": [
+            "predict",
+            "predict_interval",
+            "predict_quantiles",
+            "predict_var",
+        ]
+    }
+    flavor.save_model(
+        sktime_model=auto_arima_model,
+        path=model_path,
+        serialization_format=serialization_format,
+    )
+    loaded_pyfunc = flavor.pyfunc.load_model(model_uri=model_path)
+
+    model_predict = auto_arima_model.predict()
+    model_predict_interval = auto_arima_model.predict_interval()
+    model_predict_interval.columns = flatten_multiindex(model_predict_interval)
+    model_predict_quantiles = auto_arima_model.predict_quantiles()
+    model_predict_quantiles.columns = flatten_multiindex(model_predict_quantiles)
+    model_predict_var = auto_arima_model.predict_var()
+    model_predictions = pd.concat(
```

Comment:
Is there not a helper function to make this easier for users to do? This seems pretty brittle and confusing for an API.

Reply:
you mean a helper function to call multiple predict methods and combining the output into a single dataframe?

Reply:
sort of. Based on what you mentioned above about using the pmdarima flavor as inspiration, crafting the argument consumer portion of this flavor would be greatly simplified (both from a dev perspective in MLflow, but also for users). I think your original intuition is the right direction to go.

Reply:
not relevant anymore with revised interface